src/app/hearing/components/view-hearing/view-hearing.component.ts

```diff
@@ -105,6 +111,6 @@ export class ViewHearingComponent implements OnInit {
     }
 
     private fetchHearing() {
-        this.hearingService.getById(this.hearingId);
+        this.hearingService.getById(this.hearingId).subscribe();
```

Comment:
try to refactor it to not return observable, but just void.

Discussed offline